server/src/main/scala/org/ensime/server/Server.scala

```diff
@@ -129,7 +127,11 @@ object Server extends AkkaBackCompat {
       throw new RuntimeException(s".ensime file ($ensimeFile) not found")
 
     implicit val config: EnsimeConfig = try {
-      EnsimeConfigProtocol.parse(Files.toString(ensimeFile, Charsets.UTF_8))
+      val source = scala.io.Source.fromFile(ensimeFile, "utf-8")
```

Comment:
we definitely have something to do this in our files / paths implicit classes. `scala.io.Source` is ridiculously inefficient btw.